app/templates/events/manageLabor.html

```diff
@@ -0,0 +1,146 @@
+      </thead>
+      <tbody class="align-middle" id="laborTable">
+        <!-- {% if eventLaborData %} -->
+          {% for participant in eventLaborData %}
+            <tr>
+              <td>
+                <input class="form-control" type='hidden' name="username" id="{{participant.user.username}}" value="{{participant.user}}" />
+                <a href="/profile/{{participant.user}}">{{participant.user.firstName}} {{participant.user.lastName}}</a>
+                {% if participant.user in bannedUsersForProgram %}
+                  <a href="#" data-toggle="tooltip" data-placement="top" title="User is banned from this program.">
+                    <span class="bi bi-x-circle-fill text-danger"></span>
+                  </a>
+                {% endif %}
+              </td>
+              <td>{{participant.user.email}}</td>
+              <td>{{participant.user.phoneNumber}}</td>
+              <td style="text-align:center">
+                
+                <input type="checkbox" id = "workedCheckbox_{{participant.user.username}}" class='attendanceCheck'
+                  name="checkbox_{{participant.user.username}}"
+                  {% if participant.didWork %} 
+                  checked
+                  {% endif %} />
+
+                  
+              </td>
+              <td style="text-align:center">
+                <button type="button" class="removeLabor btn btn-danger btn-sm"
+                  id="{{participant.user.username}}" aria-label="remove labor">
+                  <span class="bi bi-trash h5 align-middle"></span>
+                </button>
+              </td>
+            </tr>
+          {% endfor %}
+        <!-- {% endif %} -->
+      </tbody>
+    </table>
+      <div class="row">
+        <div class="col"></div>
+        <button type="submit" class="btn btn-primary float-end nowrap col-auto mx-3">Save</button>
+      </div>
+  </div>
+</form>
+
+
+<!-- #################  Modal  ################# -->
+<div class="modal fade float-start" id="addLaborModal" tabindex="-1" aria-labelledby="addLaborModalLabel" aria-hidden="true">
+  <div class="modal-dialog">
+    <div class="modal-content">
+      <div class="modal-header">
+        <h5 class="modal-title">Add Student Labor</h5>
+        <button type="button" class="btn-close" data-bs-dismiss="modal" aria-label="Close"></button>
+      </div>
+      <div class="modal-body">
+        <div class="input-group">
+          <div class="form-group input-group-x form-outline ui-widget" style="width:80%;">
+            <input type="input" id="addLaborInput" class="form-control" placeholder="Search" autocomplete="off"/>
+          </div>
+          <button type="button" class="btn btn-primary btn-sm">
+            <i class="bi bi-search ms-0"></i>
+          </button>
+        </div>
+        <ul class="list-unstyled mx-3 mt-2" id= "addLaborList">
+          {% if event.isRepeating and not event.isFirstRepeatingEvent%}
+            {% if repeatingingLabor %}
```

Comment:
I believe this is a typo. It says "repeatingingLabor". I believe it should be "repeatingLabor